scripts/cdn-version.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Updates the CDN paths with the current version of NPM
+ */
+const fs = require('fs');
+const files = ['./docs/index.html', './docs/ds6/index.html'];
+const contents = files.map(file => fs.readFileSync(file, 'utf8'));
```

Comment:
Why not read the file inside the forEach loop? It saves a loop (not a big deal either way since it's only 2 files)

Reply:
LGTM overall, just a minor nit

Reply:
🤷🏻‍♂️ Quick copied code from @mlrawlings.

Like this?
```suggestion
/*
 * Updates the CDN paths with the current version of NPM
 */
const fs = require('fs');
const files = ['./docs/index.html', './docs/ds6/index.html'];

files.forEach(file => {    
    const newContents = fs.readFileSync(file, 'utf8').replace(/version\:.*\n/gi, `version: ${process.env.npm_package_version}\n`);

    fs.writeFile(file, newContents, 'utf8', (err) => {
        if (err) {
            return console.log(err);
        }
    });
});
```

Reply:
👍